hiku/query.py

```diff
@@ -309,38 +308,53 @@ def _merge(
                 yield field
 
     if not visited_fields and not to_merge:
-        for fr in fragments_iter:
+        for fr in chain.from_iterable(e.fragments for e in nodes):
```

Comment:
is it necessary to rm variable at 293 here?

Reply:
its not used anywhere but here